README.md

```diff
@@ -57,6 +57,8 @@ You also have the power to programatically set any configuration within the play
     # First time set up
     npm install -g grunt
     npm install
+    npm install webpack minimist
+    export AIR_HOME={AIRSDK_Compiler}

```

Comment:
We can add webpack and minimist to the dev dependencies in package.json.

How does `export AIR_HOME={AIRSDK_Compiler}` help? Do you already have $AIRSDK_Compiler defined in your environment? If so maybe mxmlc is already in your path ad we can make changes to support that? In the future we plan to make Flash compilation optional and only warn users if mxmlc is not found / no swf compiled.